### PR TITLE
Expose PORTS Added

### LINKED ISSUE #141: Missing EXPOSE in Dockerfile
Please add EXPOSE token to the Dockerfile with the port of the service.

### PR DESCRIPTION
**Pull request by Jagatjot**
Fixes #141

### CODE REVIEW

**Comment by Jagatjot** [comment]
@fgalan Could you please review ?

File: docker/Dockerfile [hunk 1]
```diff
@@ -93,3 +93,6 @@ ENV NODE_ENV=production
 
 ENTRYPOINT ["pm2-runtime", "bin/iotagent-ul"]
 CMD ["-- ", "config.js"]
+
+# Expose 4041 for NORTH PORT, 7896 for HTTP PORT
+EXPOSE 4041 7896
```
Comment (jason-fox) [inline docker/Dockerfile]: Don't hardcode the ports, use default values and existing `ENV`s:

```Dockerfile
# Expose IOTA_NORTH_PORT and  IOTA_HTTP_PORT
EXPOSE ${IOTA_NORTH_PORT:-4041} ${IOTA_HTTP_PORT:-7896}
```

Comment (fgalan) [reply docker/Dockerfile]: Fully agree. @Jagatjot note that we have env vars for this already documented in https://hub.docker.com/r/telefonicaiot/iotagent-ul/

```
    environment:
       ...
        - "IOTA_NORTH_PORT=4041"
        - "IOTA_HTTP_PORT=7896"
       ...
```

In addition, note that ENTRYPOINT and CMD should be the last lines in the file (is part of our style in Dockerfiles ;) so please move the EXPOSE clause up. Maybe just after `ENV NODE_ENV=production`  is a good place. @jason-fox maybe have some feedback on this.

Comment (Jagatjot) [reply docker/Dockerfile]: Fixed by https://github.com/telefonicaid/iotagent-ul/pull/342/commits/2cbfd93109345a6800f8f8877ea2f0c76167e2fd

File: docker/Dockerfile [hunk 2]
```diff
@@ -91,5 +91,8 @@ RUN \
 USER node
 ENV NODE_ENV=production
 
+# Expose 4041 for NORTH PORT, 7896 for HTTP PORT
+EXPOSE ${IOTA_NORTH_PORT} ${IOTA_HTTP_PORT}
```
Comment (jason-fox) [inline docker/Dockerfile]: This is missing the default values- currently the ports won't work without `ENV`s being passed it should read:

> 
> ```dockerfile
> # Expose IOTA_NORTH_PORT and  IOTA_HTTP_PORT
> EXPOSE ${IOTA_NORTH_PORT:-4041} ${IOTA_HTTP_PORT:-7896}
> ```

Comment (Jagatjot) [reply docker/Dockerfile]: Fixed by https://github.com/telefonicaid/iotagent-ul/pull/342/commits/02dd708ad494d1a8fd96dbfb1a43a0f107927330

Comment (jason-fox) [reply docker/Dockerfile]: LGTM

File: docker/Dockerfile (hunk 1 above)
Comment (jason-fox) [reply docker/Dockerfile]: LGTM

**Pull request merged**